sdcm/nemesis.py

```diff
@@ -3885,6 +3912,20 @@ def disrupt_run_unique_sequence(self):
         self.disrupt_grow_shrink_cluster()
         InfoEvent(message="Finished grow_shrink disruption").publish()
 
```

Comment:
We need a way to make this one work only on large partition cases

Reply:
I do not sure how this nemesis connected to the commit